### PR TITLE
Upgrade version of Quarkus to the latest stable LTS 3.2.6.Final

### PR DESCRIPTION
**Pull request by bmscomp**
Upgrading the version of Quarkus to the latest one stable TLS one 3.2.6.Final

### CODE REVIEW

**Comment by hanMue** [comment]
Can anyone please have a look at that PR?

**Comment by bmscomp** [comment]
As Quarkus moved to latest stable version I think it's better to move up the update version to 3.6.6

**Comment by gastaldi** [comment]
I'd recommend to upgrade to the latest LTS instead (3.2.6.Final).

File: pom.xml [hunk 1]
```diff
@@ -22,13 +22,13 @@
     <url>https://github.com/quarkiverse/quarkus-cucumber</url>
   </scm>
   <properties>
-    <compiler-plugin.version>3.8.1</compiler-plugin.version>
+    <compiler-plugin.version>3.11.0</compiler-plugin.version>
```
Comment (gastaldi) [inline pom.xml:25]: This is inherited from quarkiverse-parent. It can be safely removed (including the plugin declaration)

Comment (bmscomp) [reply pom.xml]: @gastaldi This is done on the last update

**Review by bmscomp — COMMENTED** [review]
Pull request updated

File: pom.xml (hunk 1 above)
Comment (gastaldi) [reply pom.xml]: Actually the declaration in the `runtime` module is necessary, just not in the pluginManagement, sorry for not being clear

**Comment by bmscomp** [comment]
@gastaldi getting ride of maven compile plugin version is failing the build so I restored it

File: pom.xml (hunk 1 above)
Comment (bmscomp) [reply pom.xml]: I just removed it and the build is failing

**Pull request merged**